xdsl/transforms/convert_stencil_to_csl_stencil.py

```diff
@@ -331,14 +313,61 @@ def get_op_split(
                         a.append(use.operation)
                         rem.remove(use.operation)
 
-    if len(a_exports) == 1:
-        return a, b + rem
+    cnst_exports = [cnst for cnst in a_exports if isinstance(cnst, arith.Constant)]
+    if len(a_exports) == 1 + len(cnst_exports):
+        recv_chunk_ops, done_exch_ops = list[Operation](), list[Operation]()
+        for op in ops:
+            if op in a:
+                recv_chunk_ops.append(op)
+                if op in cnst_exports:
+                    done_exch_ops.append(cln := op.clone())
+                    op.result.replace_by_if(
```

Comment:
nit: this function is called `get_...`, but it may modify input ops

Reply:
Renaming to `split_ops`